src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/AutoFormattingOptions.cs

```diff
@@ -21,6 +24,7 @@ namespace Microsoft.CodeAnalysis.Formatting
         bool FormatOnSemicolon,
         bool FormatOnCloseBrace)
     {
+#if !CODE_STYLE
```

Comment:
Should these methods potentially become extensions / static helpers defined at the workspaces layer, instead of in the block?

Reply:
yes.  that's also an option (potentially doing things with partials as well).  for now, i'm just keeping the general style we've had when we push stuff down.

Reply:
I think I prefer Dave's suggestion here when all methods in the type are specific to Workspaces layer. For types where few methods are shared and few specific to CodeStyle or Workspaces layer, but not both, it seems fine to keep all methods together in a single file with `#if` directives, especially when methods are related and having them in the same file adds context between them.

Reply:
that works for me.  can do in followup

Reply:
handled in https://github.com/dotnet/roslyn/pull/60462.  thanks.